resources/views/components/link-preview-card.blade.php

```diff
@@ -0,0 +1,50 @@
+<div
+    id="link-preview-card"
+    data-url="{{ $url }}"
+    class="mx-auto mt-2 min-w-full group/preview" data-navigate-ignore="true"
+>
+@if ($data->has('html'))
+    @if ($data->get('site_name') === 'X (formerly Twitter)')
+        @php($tweetId = explode('/', $url))
+        <div data-tweet-id="{{ end($tweetId) }}" class="w-full overflow-hidden">
+            {!! $data->get('html') !!}
+        </div>
+    @else
+        <div class="w-full overflow-hidden">
+            {!! $data->get('html') !!}
+        </div>
+    @endif
+@elseif ($data->has('player'))
+    <iframe
```

Comment:
is this totally safe?

Reply:
It should be safe, but I have removed it for now, in favour of the oEmbed players, as that gives us a way to ensure html is secure by having access control over which oEmbed apis are called.

Reply:
Done